bblfsh/test.py

```diff
@@ -0,0 +1,35 @@
+import socket
+import subprocess
+import time
+import unittest
+
+from bblfsh import BblfshClient
+
+
+class BblfshTests(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        subprocess.check_call(
+            "docker run --privileged -p 9432:9432 --name bblfsh -d "
+            "bblfsh/server".split())
+        sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+        result = -1
+        while result != 0:
+            time.sleep(0.1)
+            result = sock.connect_ex(("0.0.0.0", 9432))
+        sock.close()
+
+    @classmethod
+    def tearDownClass(cls):
+        subprocess.check_call("docker rm -f bblfsh".split())
+
+    def setUp(self):
+        self.client = BblfshClient("0.0.0.0:9432")
+
+    def testUAST(self):
+        uast = self.client.fetch_uast(__file__, "Python")
```

Comment:
`uast` in this case is a response object more than a UAST that would be part of it, I'm right? Also should probably check for errors in the response (`response.errors` and/or `response.status`).